docs/src/developer/installing.rst

```diff
@@ -0,0 +1,57 @@
+***************
+Getting started
+***************
+In this section, we'll guide you to install the dependencies and environment to develop on Oríon.
+We made our best to automate most of the process using Python's ecosystem goodness to facilitate
+your onboarding. Let us know how we can improve!
+
+Oríon
+=====
+The first step is to clone your remote repository from Github (if not already done, make sure to
+fork our repository_ first).
+
+.. code-block:: sh
+
+   $ git clone https://github.com/epistimio/orion.git --branch develop
+
+.. tip::
+
+   The usage of a `virtual environment`_ is recommended, but not necessary.
+
+   .. code-block:: sh
+
+      $ mkvirtualenv -a $PWD/orion orion
+
+Then, you need to deploy the project in `development mode`_ by invoking the ``setup.py`` script with
+``develop`` argument or by using ``pip install --editable``.
+
+.. code-block:: sh
+
+   $ python setup.py develop --optimize=1
```

Comment:
Is the option ``--optimize=1`` very useful? Otherwise I would favor simplicity of `python setup.py develop`.

Reply:
I don't know what it does. I guessed it was canon since it was in the original doc 😂.

I will investigate. I agree that simplicity is better!

Reply:
So it controls the level of optimization when compiling source files. I don't think having it on is critical for development activities. What do you think?

However, it would be a good idea to have the optimized binaries for packages we release on PyPI.

documentation: https://setuptools.readthedocs.io/en/latest/easy_install.html#reference-manual

Reply:
Yes, it may be better to have the compiled .pyo files available on PyPI. So if we keep it in tox.ini, we should keep this in the doc to be coherent.